docs/pages/reference/architecture/session-recording.mdx

```diff
@@ -78,6 +78,19 @@ and `proxy-sync` identically (perform synchronous recording).
 
 </Admonition>
 
+### Encryption
+
```

Comment:
To make this easier to read, I would add an example of each configuration field, showing it in context of a YAML snippet.